### PR TITLE
Re-revert: Skunkworks: Upload Lighthouse Scores to Atlas

### PR DESCRIPTION
**Pull request by mmeigs**
This is the same PR that failed last week's deployment but with all tsconfig changes removed! No other changes have happened to the PR.

Reference to [previous PR](https://github.com/mongodb/docs-worker-actions/pull/81)'s description.

More info:

Options on displaying this information still being decided. But primary objective is persisting runs to the database so that future decisions will still have past data.

### CODE REVIEW

File: dist/build-artifact/index.js
Comment (seungpark) [inline dist/build-artifact/index.js:1]: why do we have the diffs here if no change in tsconfig?

Comment (mmeigs) [reply dist/build-artifact/index.js:1]: Great question: two possible answers. I had "type": "module" in the package.json. I've just removed that.

Also, couuuld be the upgrade of mongodb, but most likely it's the statement above.

Comment (mmeigs) [reply dist/build-artifact/index.js:1]: ^^ I think it was both? I removed `"type": "module"` and deleted all the extra package.jsons that had been created for that. So, I have to believe its the mongodb upgrade? Nothing else is different.

Comment (seungpark) [reply dist/build-artifact/index.js:1]: seems like that did the trick 👍 only certain bundles are updated with version / license updates only

**Review by seungpark — APPROVED** [review]

**Pull request merged**